src/events/types/CreateEventOptions.ts

```diff
@@ -2,4 +2,6 @@ import { BaseEvent } from './BaseEvent';
 
 export type CreateEventOptions = Required<
   Pick<BaseEvent, 'scheduledTime' | 'payload' | 'destination'>
->;
+> & {
+  disableEncryption?: boolean;
```

Comment:
Same comment as below; I think this is a bit misleading and might be better as some sort of `override`

Reply:
`skipEncryptionIfEnabled`?